files/CumulusLinux-2.2

```diff
@@ -1,199 +0,0 @@
-# This is a copy of the wheezy deboostrap script
```

Comment:
So, this script is the same for all debian variants and the same for all ubuntu variants. The only difference is what the script is called. I pulled all of these scripts from the debian 8 debbootstrap package.